linera-service/src/project.rs

```diff
@@ -76,14 +76,11 @@ impl Project {
     }
 
     pub async fn test(&self) -> Result<()> {
-        if Self::runner_path().await.is_err() {
-            debug!("Linera test runner not found");
-            Self::install_test_runner()?;
-        }
+        let runner_path = util::resolve_binary(RUNNER_BIN_NAME, RUNNER_BIN_CRATE).await?;
```

Comment:
In this case, perhaps a message explaining how to install the runner?

Reply:
It's already there

Reply:
Indeed! Somehow missed it :)